src/elements/ProjectSettingsGeneral/ProjectSettingsGeneral.js

```diff
@@ -38,6 +38,7 @@ import './projectSettingsGeneral.scss'
 const ProjectSettingsGeneral = ({
   addProjectLabel,
   changeOwnerCallback,
+  dispatch,
```

Comment:
You don't need to pass `dispatch` and actions (`setNotification`) via props. Use imports for that